flight/Modules/Actuator/actuator.c

```diff
@@ -83,13 +86,18 @@ static bool manualControlCommandUpdated = true;
 // set true to ensure they're fetched on first run
 static volatile bool actuator_settings_updated = true;
 static volatile bool mixer_settings_updated = true;
```

Comment:
My main concern is that it might not let autotune still work on some F1 configurations (that are not even using triflight).